.coderabbit.yaml

```
@@ -0,0 +1,88 @@
+
+    # For Authentications and Authorization
+    - path: "app/(auth)/**/*.ts"
+      instructions: |
+        - Enforce HTTPS, secure sessions, rate limits
+        - Review cookie handling and JWT usage
+
+    - path: "app/api/**/*.ts"
+      instructions: |
+        - Validate API inputs, handle errors, and check auth
+        - Optimize DB queries and monitor for sensitive leaks
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)